internal/handlers/stats_test.go

```diff
@@ -0,0 +1,80 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func abs(a int64) int64 {
+	if a < 0 {
+		a *= -1
+	}
+	return a
+}
+
+func semiEqual(a, b, precision int64) bool {
```

Comment:
Is this going to be needed for some fuzzy date comparison or something in the future?